app/controllers/concerns/trackable.rb

```diff
@@ -8,6 +8,6 @@ def track_event(name = "", properties = {})
 
     # default if no event name is provided
     name = "Viewed #{controller_name} #{action_name}" if name.empty?
-    FuguWorker.perform_async(name, properties)
+    FuguWorker.perform_async_with_failover(name, properties)
```

Comment:
Cool, didn't know this method existed. Just assumed that it doesn't cause the app to throw an exception if Redis is down. Great!

Reply:
It actually doesn't exist in sidekiq, it's a custom method that I added in FuguWorker. Maybe I should open up a PR on the sidekiq gem and propose it